src/cli_shared/cli/config.rs

```diff
@@ -92,6 +93,25 @@ impl Default for ChainIndexerConfig {
     }
 }
 
+#[derive(Deserialize, Serialize, PartialEq, Eq, Debug, Clone)]
+#[cfg_attr(test, derive(derive_quickcheck_arbitrary::Arbitrary))]
+pub struct FeeConfig {
+    #[serde(with = "crate::lotus_json")]
+    pub max_fee: TokenAmount,
+}
+
+impl Default for FeeConfig {
+    fn default() -> Self {
+        // This indicates the default max fee for a message,
+        // The code is taken from https://github.com/filecoin-project/lotus/blob/release/v1.34.1/node/config/def.go#L39
+        Self {
+            max_fee: TokenAmount::from_atto(
+                num_bigint::BigInt::from_str("70000000000000000").unwrap(),
```

Comment:
There are more performant initialization options for `TokenAmount` rather than parsing strings. Please check the `TokenAmount` interface.